templates/OperationSnippet.stencil

```diff
@@ -10,18 +10,42 @@ public func {{ op.name }}(
     guard let url = self.url(forTemplate: urlTemplate) else { return }
 
     // Construct query
+    {% if op.queryInitParams.count > 0 %}
     var queryParams: [QueryParameter] = [
-        {% for param in op.queryParams %}
-        "{{ param.key }}" : {{ param.value}}
+        {% for param in op.queryInitParams %}
+        ("{{ param.key }}" , {{ param.value}})
         {% endfor %}
     ]
+    {% elif op.queryParams.count > 0 %}
+    var queryParams = [QueryParameter]()
+    {% endif %}
```

Comment:
We want to avoid having conditional logic in our templates. I think it is totally fine to only the second version, and to simply add things after the fact.

Reply:
Removed.